hack/test_delete_ns.sh

```diff
@@ -47,6 +47,14 @@ function test_delete_ns(){
         echo "Ignoring webhook on k8s where we don't have OLM based validating webhooks"
     fi
 
+    # TODO: workaround for https://bugzilla.redhat.com/show_bug.cgi?id=2037312
+    # remove once fixed
+    echo "Explictly disabling CommonBootImageImport to be able to safely remove the product"
+    ${CMD} patch hco -n kubevirt-hyperconverged kubevirt-hyperconverged --type=json kubevirt-hyperconverged -p '[{ "op": "replace", "path": /spec/enableCommonBootImageImport, "value": false }]'
```

Comment:
The FG jsonpath is wrong...

```suggestion
    ${CMD} patch hco -n kubevirt-hyperconverged kubevirt-hyperconverged --type=json kubevirt-hyperconverged -p '[{ "op": "replace", "path": /spec/featureGates/enableCommonBootImageImport, "value": false }]'
```